### ISSUE TITLE
Reduce walk animation height

### ISSUE THREAD

**Issue by TimvdLippe** [tcmg/adventure-issues#62]
**Is your feature request related to a problem? Please describe.**
When initially walking around, I got a bit nauseated by the walk animation. The walk animation appears to be a "jump to a different tile" animation. Especially the jump to "up" is particularly strong.

**Describe the Idea or Solution you'd like**
Ideally, we would reduce the amount the character jumps, to reduce the overall animation height. For example, when moving up, we reduce the pixel height to what the character jumps to, which currently seems equivalent to tile above it. E.g. the current animation is 1. current tile 2. tile above tile to walk to 3. the tile to walk to. Ideally we reduce step 2 to be "just a bit above the tile to walk to, to reduce the jump height.

**Why?**
When opening the game and walking around, I initially experienced nausea. Over time it reduced a bit, but even after walking around for a bit, I can't dismiss the nausea feeling. My suspicion that the feeling is coming from the jumpiness of the character, but this is an assumption on my end.

**Comment by tcmg** [comment]
I don't think we really have time to adjust this but if you're interested in submitting a PR it'd be great to get your help on the project! Maybe there can be a settings toggle for "reduced motion".

**Comment by TimvdLippe** [comment]
@tcmg Could you send me the details on where I can contribute to my work email? Please CC my manager in it.
